cmd/bootstrap/run/qc_test.go

```diff
@@ -28,8 +28,32 @@ func TestGenerateRootQC(t *testing.T) {
 	votes, err := GenerateRootBlockVotes(&block, participantData)
 	require.NoError(t, err)
 
-	_, err = GenerateRootQC(&block, votes, participantData, participantData.Identities())
+	_, invalid, err := GenerateRootQC(&block, votes, participantData, participantData.Identities())
 	require.NoError(t, err)
+	require.Len(t, invalid, 0) // no invalid votes
+}
+
+func TestGenerateRootQCWithSomeInvalidVotes(t *testing.T) {
+	participantData := createSignerData(t, 10)
+
+	block := unittest.BlockFixture()
+	block.Payload.Guarantees = nil
+	block.Payload.Seals = nil
+	block.Header.Height = 0
+	block.Header.ParentID = flow.ZeroID
+	block.Header.View = 3
+	block.Header.PayloadHash = block.Payload.Hash()
```

Comment:
```suggestion
	block := unittest.GenesisFixture()
```

Could we do this instead? I think everything is the same except the view.